pkg/controllers/status_cluster.go

```diff
@@ -209,6 +209,9 @@ func getComponentDeployment(ctx context.Context, key client.ObjectKey, component
 }
 
 func (r *MilvusStatusSyncer) UpdateIngressStatus(ctx context.Context, mc *v1beta1.Milvus) error {
+	if mc.Spec.Com.Proxy.Ingress == nil {
```

Comment:
We combined the `MilvusCluster` & the `Milvus` in `v1beta1`,  so `mc.Spec.Com.Standalone.Ingress` will have the same issue, can you add that, too?

Reply:
It has been handled well in v1beat1.
https://github.com/milvus-io/milvus-operator/blob/618f40a1b98c9e70174e73bdf4963073aabcf97f/pkg/controllers/ingress.go#L17-L30

Reply:
https://github.com/milvus-io/milvus-operator/blob/618f40a1b98c9e70174e73bdf4963073aabcf97f/apis/milvus.io/v1beta1/milvus_types.go#L34-L37